CRM/Eventsextras/Form/Settings.php

```diff
@@ -72,9 +69,9 @@ public function postProcess() {
     $valuesToSave = array_intersect_key($submittedValues, $configFields);
     $result = civicrm_api3('setting', 'create', $valuesToSave);
     $session = CRM_Core_Session::singleton();
-    if($result['is_error']== 0){
+    if( $result['is_error']== 0){
```

Comment:
Space immediately after the IF is what I meant actually i.e `if ($result['is_error']== 0){`